lib/KCard/index.vue

```diff
@@ -246,6 +271,25 @@
             },
           };
         }
+        if (
```

Comment:
Just from code it seems to me that ` thumbnailAlignClass` is taking care of everything already. Is this whole `if` section possibly a leftover from the first PR version that just needs to be removed?